networking/driver/sanity/runtest.sh

```diff
@@ -2,6 +2,8 @@
 
 . ../../../cki_lib/libcki.sh || exit 1
 
+TEST=$(egrep '^name=' metadata | awk -F'=' '{print $NF}')
```

Comment:
Hi @idorax is there a reason your adding this variable  here ? Seems like it's only needed for the common library ? Should we remove it here or other places where common lib is not a dep ?

Reply:
Hi @rasibley , it is because `TEST` var was defined in `Makefile`, once Makefile is removed, I'd like to define it in all runtest.sh. For more about `TEST=` in Makefiles, please refer to:
o http://pastebin.test.redhat.com/867415

Reply:
TEST was just used to generate the Makefile metadata,  so I don't think it's needed unless the test is actually referencing that var elsewhere. Maybe cbouchar@redhat.com can confirm ? More details here:  https://restraint.readthedocs.io/en/latest/remove_rhts.html

``` 
export TEST=/examples/no-rhts/sample-before
export TESTVERSION=1.0

BUILT_FILES=

FILES=$(METADATA) runtest.sh Makefile PURPOSE

.PHONY: all install download clean

run: $(FILES) build
        ./runtest.sh

build: $(BUILT_FILES)
        test -x runtest.sh || chmod a+x runtest.sh

clean:
        rm -f *~ $(BUILT_FILES)

include /usr/share/rhts/lib/rhts-make.include

$(METADATA): Makefile
        @echo "Owner:           User ABC1 <userabc1@example.com>" > $(METADATA)
        @echo "Name:            $(TEST)" >> $(METADATA)
        @echo "TestVersion:     $(TESTVERSION)" >> $(METADATA)
        @echo "Path:            $(TEST_DIR)" >> $(METADATA)
        @echo "Description:     Sample-before-no-rhts" >> $(METADATA)
        @echo "Type:            Sanity" >> $(METADATA)
        @echo "TestTime:        5m" >> $(METADATA)
        @echo "Priority:        Normal" >> $(METADATA)
        @echo "License:         GPLv2+" >> $(METADATA)
        @echo "Confidential:    no" >> $(METADATA)
        @echo "Destructive:     no" >> $(METADATA)
        @echo "Releases:        -RHEL7 -RHEL8" >> $(METADATA)
        @echo "Architectures:   x86_64" >> $(METADATA)

```

Reply:
Hi @rasibley, thanks for the clarification. I have removed `TEST=...` from all runtest.sh, please take a look again, thank you!

- http://pastebin.test.redhat.com/867485

Reply:
Thanks @idorax just trying to simplify things ,but lets have the bot test just in case I was wrong :-) @cki-bot please test once more.